### DISCUSSION TITLE
Champion

### DISCUSSION THREAD

**Discussion by morteeke** [brittonhayes/aos#21] [General]
Hey @brittonhayes ,

I know you have changed the model to accept a champion value but it seems you haven't added it to units.yaml?
Do I just add it myself?

Thanks!

**Comment by brittonhayes** [comment]
Hey again @morteeke! Yeah feel free to open a pull request adding the champion.

The model accepts the champion as either blank or with a string value so yep, just add that field to the units.yaml with any data you'd like to add.

**Comment by brittonhayes** [comment]
Any help adding data is always appreciated because it enables me to keep focusing on stability, features, tests etc.

This weekend I rolled out a few features that I think you'll get some great use out of.

- GraphQL
- GraphQL Apollo Playground
- Limits and Offsets support in query params
- A flashy new landing page for the API website with a link to the docs and GraphQL playground

If you haven't yet, I'd go checkout the GraphQL playground since you mentioned you're making something for mobile and this gives you a lot of flexibility to only request the data you need.

[New website homepage](https://aos-api.com/)

[Example GraphQL Query](https://aos-api.com/graphql?explorerURLState=N4IgJg9gxgrgtgUwHYBcQC4QEcYIE4CeABAKIAeAhnAA4A2CAiroUcADpJFExICWKAZwAUAM160U%2BdKyRUE0tiAAyERQF8AlKw5cusxDt2J6CAOoIK1CEgHbOuvXMNc1h10ncg1QA)

**Reply by morteeke** [reply]
Awesome!

I have some data ready but haven't commit it yet. Will do very soon though.

**Reply by brittonhayes** [reply]
Sweet! Excited to see it. 

Just giving a heads up, I've added schemas for each of the YAML fixtures in the latest commit to main. This should improve developer experience in your text editor by showing the required fields inline as you edit the data.

- [Schemas](https://github.com/brittonhayes/aos/tree/main/schema)
- [Enabled Schemas in YAML](https://github.com/brittonhayes/aos/blob/main/fixtures/alliances.yaml#L1)

![image](https://github.com/brittonhayes/aos/assets/46035482/e6e7c110-4b0f-49d1-9c8a-29eb6dbfb3e0)

**Reply by morteeke** [reply]
That's beautiful!

**Reply by brittonhayes** [reply]
Hey again @morteeke! Which data were you adding? I was gonna add some content but didn't wanna duplicate your work.😊

I'm planning on adding some Warscrolls if you're doing units.

**Reply by morteeke** [reply]
hey @brittonhayes ,

Sorry for the late answer. I have been doing some stormcast so far. (Vanguard-raptors with longstrikes, annihilators, praetors, aetherwings)

If you already have their data, feel free to keep that data.

Thanks for asking!

**Reply by brittonhayes** [reply]
No problem at all! Doesn't overlap with anything I'm working on, so carry on! 🏃
